### ISSUE TITLE
Crashes on startup

### ISSUE THREAD

**Issue by Spaxey** [elix-x/Forever-Enough-Items#51]
---- Minecraft Crash Report ----

WARNING: coremods are present:
  MalisisCorePlugin (malisiscore-1.8.9-2.1.6.jar)
  MalisisSwitchesPlugin (malisisswitches-1.8.9-1.0.1.jar)
Contact their authors BEFORE contacting forge

// Hey, that tickles! Hehehe!

Time: 10/23/16 3:28 PM
Description: There was a severe problem during mod loading that has caused the game to fail

net.minecraftforge.fml.common.LoaderException: java.lang.ExceptionInInitializerError
    at net.minecraftforge.fml.common.LoadController.transition(LoadController.java:162)
    at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:559)
    at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:243)
    at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:417)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:329)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.ExceptionInInitializerError
    at java.lang.Class.forName0(Native Method)
    at java.lang.Class.forName(Class.java:260)
    at mezz.jei.util.AnnotatedInstanceUtil.getInstances(AnnotatedInstanceUtil.java:28)
    at mezz.jei.util.AnnotatedInstanceUtil.getModPlugins(AnnotatedInstanceUtil.java:19)
    at mezz.jei.ProxyCommonClient.preInit(ProxyCommonClient.java:59)
    at mezz.jei.JustEnoughItems.preInit(JustEnoughItems.java:55)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:560)
    at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:211)
    at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:189)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:118)
    at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:556)
    ... 10 more
Caused by: net.minecraftforge.fml.relauncher.ReflectionHelper$UnableToFindFieldException: java.lang.NoSuchFieldException: uidCacheEnabled
    at net.minecraftforge.fml.relauncher.ReflectionHelper.findField(ReflectionHelper.java:90)
    at code.elix_x.excore.utils.reflection.AdvancedReflectionHelper$AField.<init>(AdvancedReflectionHelper.java:153)
    at code.elix_x.mods.fei.client.jeioverride.JeiReflector.<clinit>(JeiReflector.java:49)
    ... 42 more
Caused by: java.lang.NoSuchFieldException: uidCacheEnabled
    at java.lang.Class.getDeclaredField(Class.java:2062)
    at net.minecraftforge.fml.relauncher.ReflectionHelper.findField(ReflectionHelper.java:81)
    ... 44 more
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- System Details --
Details:
    Minecraft Version: 1.8.9
    Operating System: Windows 7 (amd64) version 6.1
    Java Version: 1.8.0_25, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 162925832 bytes (155 MB) / 306704384 bytes (292 MB) up to 1060372480 bytes (1011 MB)
    JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP 9.19 Powered by Forge 11.15.1.1722 13 mods loaded, 13 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCH mcp{9.19} [Minecraft Coder Pack](minecraft.jar) 
    UCH FML{8.0.99.99} [Forge Mod Loader](forge-1.8.9-11.15.1.1722.jar) 
    UCH Forge{11.15.1.1722} [Minecraft Forge](forge-1.8.9-11.15.1.1722.jar) 
    UCH BiomesOPlenty{3.0.0} [Biomes O' Plenty](BiomesOPlenty-1.8.9-3.0.0.1927-universal.jar) 
    UCH cfm{4.0.1} [MrCrayfish's Furniture Mod](cfm-4.0.1-mc1.8.9.jar) 
    UCH excore{1.5.3} [EXCore](Excore-1.5.3-1.8.9 %281%29.jar) 
    UCE JEI{2.24.3.127} [Just Enough Items](Just-Enough-Items-Mod-1.8.9.jar) 
    UCE FEI{1.0.10} [Forever Enough Items](ForeverEnoughItems-1.0.10-1.8.9.jar) 
    UCH malisiscore{1.8.9-2.1.6} [MalisisCore](malisiscore-1.8.9-2.1.6.jar) 
    UCH malisisdoors{1.8.9-3.1.3} [MalisisDoors](malisisdoors-1.8.9-3.1.3.jar) 
    UCH malisisswitches{1.8.9-1.0.1} [Malisis Switches](malisisswitches-1.8.9-1.0.1.jar) 
    UCH RadixCore{1.8.9-2.1.0} [RadixCore](RadixCore-1.8.9-2.1.0-universal.jar) 
    UCH MCA{1.8.9-5.1.1.2} [Minecraft Comes Alive](MCA-1.8.9-5.1.1.2-universal.jar) 
    Loaded coremods (and transformers): 
MalisisCorePlugin (malisiscore-1.8.9-2.1.6.jar)
  net.malisis.core.util.chunkcollision.ChunkCollisionTransformer
  net.malisis.core.util.chunkblock.ChunkBlockTransformer
  net.malisis.core.renderer.transformer.MalisisRendererTransformer
  net.malisis.core.renderer.icon.asm.TextureMapTransformer
MalisisSwitchesPlugin (malisisswitches-1.8.9-1.0.1.jar)
  net.malisis.switches.asm.MalisisSwitchesTransformer
    GL info: ' Vendor: 'Intel' Version: '4.0.0 - Build 10.18.10.4276' Renderer: 'Intel(R) HD Graphics 4000'

**Comment by Elix-x** [comment]
Duplicate of #50.

TLDR: Will be fixed in 1.0.17.

**Comment by Spaxey** [comment]
Cool. When do you think that might be out?

**Comment by DatrixTHLK** [comment]
Possibly in a few days or whenever he gets to his work space. Make sure to use gist.github.com when pasting a crash log or pastefs.com or pastebin.com. It will save on unnecessary space used in the github forum

**Comment by Elix-x** [comment]
So i fixed bugs reported on github and then found a few others.
I will not be able to release .17 today, but i guarantee that it will be released tomorrow.
Thanks for your patience.

**Comment by Spaxey** [comment]
Do you have the update for 1.8.9...? I can't find it.

**Comment by Elix-x** [comment]
Sadly, i don't support 1.8 anymore.

**Comment by Spaxey** [comment]
Why not..?

**Comment by Elix-x** [comment]
Most of 1.8.9 mods are already on 1.10.2. And 1.8.9 is still different enough that maintaining it in parallel would (almost) double the time required (for me).

Yes, thaumcraft, buildcraft, AoD..., i know, that's why most.
But they will come to 1.10.2 eventually.
